### ISSUE TITLE
link rel="self" broken for Shaarli in subdir

### ISSUE THREAD

**Issue by xrat** [shaarli/Shaarli#694]
I think the following code from `application/FeedBuilder.php`
```
$pageaddr = escape(index_url($this->serverInfo));
(...)
$data['self_link'] = $pageaddr . escape(ltrim($this->serverInfo['REQUEST_URI'], '/'));
```
causes `<atom:link rel="self"` (as in RSS feeds) and `<link rel="self"` (as in ATOM feeds) to list subdirectories twice if Shaarli is installed in a sub(sub?) directory. For 2 of my installations which are in /subdir/shaarli I get for instance 
```
<atom:link rel="self" href="http://example.net/subdir/shaarli/subdir/shaarli/?do=rss"  />
```

**Comment by ArthurHoaro** [comment]
This should have been fixed in #662 which hasn't been release yet. Can you confirm that you use the last release?

**Comment by xrat** [comment]
Ooops, sorry I missed that this was already fixed. Code works fine. Thanks a lot!
